app/assets/javascripts/components/util/routes.jsx

```diff
@@ -23,15 +24,16 @@ const routes = (
     <Route path="/courses/:course_school/:course_title" component={Course} />
     <Route path="/course_creator" component={ConnectedCourseCreator} />
     <Route path="/users/:username" component={UserProfile} />
-    <Route path="/campaigns/:campaign_slug/overview" component={CampaignOverviewHandler} />
+    {/* <Route path="/campaigns/:campaign_slug/overview" component={CampaignOverviewHandler} /> */}
     <Route path="/campaigns/:campaign_slug/alerts" component={CampaignAlerts} />
-    <Route path="/campaigns/:campaign_slug/ores_plot" component={CampaignOresPlot} />
+    {/* <Route path="/campaigns/:campaign_slug/ores_plot" component={CampaignOresPlot} /> */}
```

Comment:
These can be deleted rather than just commented out... once we have the new versions working, no reason to save the old code.

Reply:
I removed those commented lines.